frontend/src/scenes/events/EventsTable.js

```diff
@@ -190,10 +160,110 @@ function _EventsTable({ fixedFilters, filtersEnabled = true, pageKey }) {
             },
         },
     ]
+    const otherEventProperties = eventProperties.filter(
+        (e) => defaultColumns.find((d) => d.eventProperties && d.eventProperties.includes(e.value)) === undefined
+    )
+    const availableConfigsOptions = [
+        ...defaultColumns.map(tableColumnToCheckboxOption),
+        ...otherEventProperties.map(eventPropertyToCheckboxOption),
+    ]
+    let selectedConfigOptions = columnConfig === 'DEFAULT' ? defaultColumns.map((e) => e.key) : columnConfig
+
+    const columnsToRenderFromConfig =
+        columnConfig === 'DEFAULT'
+            ? defaultColumns
+            : columnConfig.map(
+                  (e) =>
+                      defaultColumns.find((d) => d.key === e) || {
+                          title: keyMapping['event'][e] ? keyMapping['event'][e].label : e,
+                          key: e,
+                          render: function renderURL({ event }) {
+                              if (!event) {
+                                  return { props: { colSpan: 0 } }
+                              }
+                              if (filtersEnabled) {
+                                  return (
+                                      <FilterPropertyLink
+                                          className={'ph-no-capture ' + rrwebBlockClass}
+                                          property={e}
+                                          value={event.properties[e]}
+                                          filters={{ properties }}
+                                      />
+                                  )
+                              }
+                              return <Property value={event.properties[e]} />
+                          },
+                          ellipsis: true,
+                      }
+              )
+    let columns = [
+        {
+            title: `Event${eventFilter ? ` (${eventFilter})` : ''}`,
```

Comment:
This column is now super squashed, whereas some others (like OS or Usage) are way too wide.

![image](https://user-images.githubusercontent.com/1727427/104178639-fa427e80-540a-11eb-8ee3-7e6256d2baee.png)

Reply:
Shall I just increase width of event or make all columns width fluid instead of fix width?

Reply:
Fixed it by adding min-width and max width to columns in ca9c8530c0ada3d91b60183cdd24520bc0ce2363